io.github.xplshn.Openarena.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.xplshn.Openarena
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+# CHANGED: Let it look up the command via $PATH
+command: launch
+modules:
+  # CHANGED: Module name isn't displayed to users and convention says lowercase
+  - name: openarena
+    buildsystem: simple
+    sources:
+      # CHANGED: Pull direct from the upstream source
+      - type: git
+        url: "https://github.com/ec-/Quake3e"
+        commit: "f69865508ef4f3d63bcc82c8505d661a1c290f92"
+        tag: "2022-10-14"
+        x-checker-data: # Allow bot to automatically make PRs for new releases
+          type: git
+          tag-pattern: "^(\\d{4}-\\d{2}-\\d{2})$"
+    # CHANGED: Removed config-opts because it's for ./configure, not bare make
+    build-commands:
+      - make USE_CODEC_VORBIS=1 USE_INTERNAL_OGG=1 USE_VOIP=1 USE_CODEC_OPUS=1 USE_OPENAL=1 USE_MUMBLE=1 USE_FREETYPE=1 USE_INTERNAL_LIBS=1 BUILD_SERVER=0COPYDIR=/app/share/openarena COPYBINDIR=/app/bin DEFAULT_BASEDIR=/app/share/openarena USE_CURL=1 USE_CURL_DLOPEN=0
+      - make install "DESTDIR=${FLATPAK_DEST}/bin"
+      # CHANGED: Use ${FLATPAK_DEST}/bin because Quake3e is one of those weird
+      #          exceptions that doesn't follow platform conventions.
+  - name: openarena-data
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: "https://downloads.sourceforge.net/project/oarena/openarena-0.8.8.zip"
+        sha256: "5a8faf7f5b51f351b0a1618c06b6b98a5f1a6758f1d39818de2c87df2a0bac4a"
+    build-commands:
+      - mv baseoa missionpack /app/bin/
+  - name: launch-script
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: "https://github.com/xplshn/openarena/blob/main/launch.zip?raw=true"
```

Comment:
Any reason you're not including this file directly in Flathub's repo?

Reply:
What do you mean? I'm new to this, that file only has the .desktop. and appdata.xml + the icon

Reply:
Is there a way to include the external files on the recipe itself?

Reply:
See https://docs.flatpak.org/en/latest/first-build.html#add-a-manifest

Reply:
> See https://docs.flatpak.org/en/latest/first-build.html#add-a-manifest

Didn't realize, sorry. Anyway, the launch.zip file contains the .desktop, appdata.xml, and icon + a "script" that will launch the engine with the arguments necessary for running OpenArena